BoyumFoosballStats/Pages/PlayerDashboard/Models/PlayerDashboardViewModel.cs

```diff
@@ -3,93 +3,181 @@
 using System.Globalization;
 using System.Linq;
 using System.Threading.Tasks;
+using BoyumFoosballStats.Components.Charts.Models;
 using BoyumFoosballStats.Services.Interface;
 using BoyumFoosballStats.Shared.DbModels;
 using MudBlazor;
+using MudBlazor.Extensions;
 
 namespace BoyumFoosballStats.Pages.PlayerDashboard.Models;
 
 public class PlayerDashboardViewModel : IPlayerDashboardViewModel
 {
     private readonly IMatchCrudService _matchCrudService;
+    private readonly IPlayerCrudService _playerCrudService;
     private List<Match> Matches;
 
-    public PlayerDashboardViewModel(IMatchCrudService matchCrudService)
+    public PlayerDashboardViewModel(IMatchCrudService matchCrudService, IPlayerCrudService playerCrudService)
     {
         _matchCrudService = matchCrudService;
+        _playerCrudService = playerCrudService;
+        Players = new List<Player>();
         Matches = new List<Match>();
-        WeekChartData = new List<WeekChartDataItem>();
+        WinRateByWeekChartData = new List<ChartDataItem>();
     }
 
     public string? PlayerId { get; set; }
-    public List<WeekChartDataItem> WeekChartData { get; private set;}
-    
+    public Player? SelectedPlayer { get; set; }
+    public List<ChartDataItem> WinRateByWeekChartData { get; private set; }
+    public List<ChartDataItem> MatchesWeekChartData { get; private set; }
+    public List<ChartDataItem> WinRateByDayChartData { get; private set; }
+    public int RankingsColumnLg => SelectedPlayer is null ? 12 : 4;
+    public int RankingsColumnXs => 12;
+    public List<Player> Players { get; private set; }
+
     public async Task InitializeAsync()
     {
         Matches = (await _matchCrudService.GetAllAsync()).ToList();
-        DisplayWinRateChart();
+        var players = (await _playerCrudService.GetAllAsync()).ToList();
+        Players = players
+            .OrderByDescending(x => x.Active)
```

Comment:
I think this might be alright for now - but I think ideally we should show them where they belong in the table, but have a toggle and exclude them by default.